app/controllers/concerns/controller_exception_handling.rb

```diff
@@ -57,8 +59,8 @@ def process(exception:, controller:, **_)
 
         def message(exception:)
           message = "\n#{exception.class} (#{exception.message}):\n"
-          message << exception.annoted_source_code.to_s if exception.respond_to?(:annoted_source_code)
-          message << '  ' << trace(exception: exception).join("\n  ")
+          message = "#{message}#{exception.annoted_source_code}" if exception.respond_to?(:annoted_source_code)
+          "#{message} #{trace(exception: exception).join("\n  ")}"
```

Comment:
Could we keep separating message and trace with two space characters here, as before, for consistency in logs?

Reply:
good spot!